corehq/ex-submodules/phonelog/reports.py

```diff
@@ -328,9 +329,8 @@ def _create_rows(self, logs, matching_id=None):
 
         self.total_records = logs.count()
         for log in logs.order_by(self.ordering)[paged]:
-            date = str(log.date)
-            date_fmt = tz_utils.string_to_pretty_time(
-                date, self.timezone, fmt="%b %d, %Y %H:%M:%S")
```

Comment:
Yeah right? There were a few places during this little adventure where I saw stuff like this—one of the big motivations for making these functions stricter about what they accept and specifically less fast-and-loose with string-formatted dates as inputs.